.github/workflows/cd-static-cdn.yml

```diff
@@ -1,7 +1,7 @@
 name: CD - Static Assets
 
 on:
-  push:
+  pull_request:
```

Comment:
This was only changed to trigger a deployment in PR for testing.  Once succeeds, I will revert this change.